code/mob/living/critter/flockcritter_parent.dm

```diff
@@ -19,24 +19,24 @@
 	// HEALTHS
 	var/health_brute = 1
 	var/health_burn = 1
-	// no medicine for radio birds
+
 	metabolizes = FALSE
 	//base compute provided
 	var/compute = 0
 	// if we're extinguishing ourselves don't extinguish ourselves repeatedly
-	var/extinguishing = 0
+	var/extinguishing = FALSE
 	// FLOCK-SPECIFIC STUFF
 	var/datum/flock/flock
-	// this body sucks i want a different one
+
 	var/mob/living/intangible/flock/controller = null
 	// do i pay for building?
-	var/pays_to_construct = 1
-	// AI STUFF
-	is_npc = 1
+	var/pays_to_construct = TRUE
+
+	is_npc = TRUE
 
-	use_stamina = 0 //haha no
+	use_stamina = FALSE
 
-	can_lie = 0 // no rotate when dead
```

Comment:
useful comment

Reply:
Idk, not sure if it's needed when can_lie already does a whole lot of other stuff